### PR TITLE
feat: added home page for v-library

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
-added dark light mode
-made it responsive
image
image

### CODE REVIEW

**Comment by ajit-sharmaGH** [comment]
-added dark light mode
-made it responsive
image
image

File: src/components/home/home.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        
+          {MustWatch.map(({img, title, _id,date, caption})=>{
+            return(
+              <div key={_id} >
+                
+               <img src ={img} alt="images" className="must_watch-img"/>
+               <p>{date} </p>
+               <p >{caption} </p>
+                <h3 className="fw-600">{title}</h3>
+              </div>
```
Comment (Mohasin-Haque) [inline src/components/home/home.js:40]: Remove white spaces

**Review by Mohasin-Haque — COMMENTED** [review]
Folder structure and code is good. You can go ahead

**Review by Kothalkarpriya — APPROVED** [review]
COde is clean and the file structure is also good.

**Pull request merged**